src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.LdTokenResultLookup.cs

```diff
@@ -363,7 +363,7 @@ private unsafe bool TryGetDynamicRuntimeMethodHandleComponents(RuntimeMethodHand
 
             DynamicMethodHandleInfo* methodData = (DynamicMethodHandleInfo*)runtimeMethodHandleValue.ToPointer();
             declaringTypeHandle = *(RuntimeTypeHandle*)&(methodData->DeclaringType);
-            genericMethodArgs = null;
+            genericMethodArgs = new RuntimeTypeHandle[0];
```

Comment:
Might want to replace this with `Array.Empty<RuntimeTypeHandle>` to avoid the allocation. Do you remember where you hit this?

Reply:
Good point. I believe I hit this issue when I called `ToMethodDesc` on `TypeSystemExtensions` in my execution strategy. `genericMethodArgs` is an `out` parameter and although it's always `null` we unconditionally try to access the `Length` property here: https://github.com/dotnet/corert/blob/master/src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeSystemExtensions.cs#L148. I was always getting an access violation exception